src/index.js

```diff
@@ -1,28 +1,117 @@
 import './index.css';
 
-const tasks = [
-  { description: 'Learn Webpack', completed: true, index: 0 },
-  { description: 'Finish quiz', completed: false, index: 1 },
-  { description: 'Learn supplication', completed: false, index: 2 },
-  { description: 'Wash utensils', completed: true, index: 3 },
-];
+let tasks = [];
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+function addTask(description) {
```

Comment:
- [ OPTIONAL] I suggest you consider using arrow functions for all your functions whenever possible. They are concise and easy to read and have an implicit return. 🤔